examples/validate_config.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+#
+# Configuration validation helper - EXAMPLE VERSION
```

Comment:
I'm not clear what the purpose of this is  - it's not different than the "real" one, best I can tell.

Plus as discussed elsewhere I don't think it should be different. The program needs to validate the config anyway, so just have a --validate-config option.